word.rb

```diff
@@ -0,0 +1,12 @@
+# pull word from dictionary.txt
+class Word
+  attr_reader :game_word
+
```

Comment:
I really like the idea of having a separate class to handle the word. If you wanted to expand this functionality, you might make this class keep track of guessed letters and the hidden version of the word as well. Then the `User` class would just be concerned with interacting with the user, not with game logic.